src/operator/contrib/ngraph.cc

```diff
@@ -0,0 +1,315 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2018 Intel Corporation
+ * \file ngraph.cc
+ * \brief ngraph subgraph property for mxnet
+*/
+
+#if MXNET_USE_NGRAPH
+#include <mxnet/ndarray.h>
+#include <ngraph_graph.h>
+#include <ngraph_imperative.h>
+#include <ngraph_nnvm_ops.h>
+#include <ngraph_sgcompiler_utils.h>
+#include <ngraph_utils.h>
+
+#include "../subgraph/common.h"
+#include "../subgraph/subgraph_property.h"
+#include "./ngraph-inl.h"
+
+namespace mxnet {
+namespace op {
+
+std::shared_ptr<ngraph_bridge::Graph> get_ngraph(const NodeAttrs &attrs) {
```

Comment:
Do I understand correctly that `ngraph_bridge::Graph` is specified designed for MXNet?

Reply:
yes, this class/object is designed for NNVM/MXNet.